cloud/amazon/ec2_elb_lb.py

```diff
@@ -360,6 +360,29 @@
 except ImportError:
     HAS_BOTO = False
 
+import time
+import random
+
+def _throttleable_operation(max_retries):
+    def _operation_wrapper(op):
+        def _do_op(*args, **kwargs):
+            retry = 0
+            while True:
+                try:
+                    return op(*args, **kwargs)
+                except boto.exception.BotoServerError, e:
+                    if retry < max_retries and e.code in \
+                            ("Throttling", "RequestLimitExceeded"):
+                        retry = retry + 1
+                        time.sleep(min(random.random() * (2 ** retry), 300))
+                        continue
+                    else:
+                        raise
```

Comment:
A bare "raise" like this in an exception handler should re-raise the original exception, as I recall: https://docs.python.org/2/tutorial/errors.html#raising-exceptions